doc/man1/openssl-s_server.pod.in

```diff
@@ -613,6 +616,12 @@ Protocol names are printable ASCII strings, for example "http/1.1" or
 "spdy/3".
 The flag B<-nextprotoneg> cannot be specified if B<-tls1_3> is used.
 
+=item B<-sendfile>
+
+With KTLS enabled, if this option is set, SSL_sendfile will be used
+instead of BIO_write to pass the file requested by http. This option
+is only available where B<-WWW> or B<-HTTP> is specified.
```

Comment:
I suggest to add parenthesis after the function names as `SSL_sendfile()` and `BIO_write()`. But

Reply:
And the whole paragraph might need a polishing. How about:

~~~
If this option is set and KTLS is enabled, SSL_sendfile() will be used instead
of BIO_write() to send the HTTP response requested by a client. This option
is only valid if B<-WWW> or B<-HTTP> is specified.
~~~

Reply:
Yes, i'll polish it up